cypress/integration/header-control-menu/menu-plik.spec.js

```diff
@@ -1,23 +1,41 @@
 /// <reference types="cypress" />
+const fs = require("fs");
 
 context("Menu 'plik'", () => {
   beforeEach(() => {
     cy.visit(`${Cypress.env("baseUrl")}`);
+    cy.contains("Plik").click();
   });
 
   describe("File button", () => {
     it("Should open options menu", () => {
-      cy.contains("Plik").click();
       cy.contains("Wczytaj");
       cy.contains("Zapisz jako...");
     });
   });
 
   describe("Load schedule", () => {
     it("Should be able to load and show the schedule", () => {
-      cy.contains("Plik").click();
       cy.get('[data-cy="file-input"]').attachFile("example.xlsx");
       cy.contains("Dni miesiąca");
     });
   });
+
+  describe("Export schedule and load exported", () => {
+    it("Should be able to export and load exported schedule", () => {
+      cy.get('[data-cy="file-input"]').attachFile("example.xlsx");
+      cy.contains("Zapisz jako...").click();
+      cy.exec('whoami').then((result) => {
+        const user = result.stdout;
+        cy.readFile(`/Users/${user}/Downloads/grafik.xlsx`).then((file) =>{
```

Comment:
To jest mega duże niedopatrzenie z mojej strony, że nie da się wybrać ścieżki.
Muszę to poprawić, i spróbuję to zrobić na podstawowym poziomie ( przekazujesz folder do którego chcesz zapisać harmonogram) już w tym tygodniu

Reply:
Poprawię wtedy też ten test